docs/modules/ROOT/pages/index.adoc

```diff
@@ -768,26 +773,226 @@ sequenceDiagram
++
+.Example:
+[source,json]
+----
+"credentialSubject": {
+  ...
+  "person": [
+    {
+      "name": "First-Name Last-Name",
+      "verifiedBy": 
+      {
+        "id": "https://example-id-verifier.com",
+        "name": "Example Id Verifier",
+        "proof": "https://example-id-verifier.com/proofs/1"
+      }      
+    }
+  ]
+  ...
+}
+---- 
+[#vc-credentialsubject-person-name]
+Person name::
+The `person[?].name` property MUST be present and MUST not be empty. The `person[?].name` property defines the _<<_named_actor,named actor’s>>_ name as it was verified by the _<<Identity verifier,identity verifiers>>_.
+
+[#vc-credentialsubject-person-verifiedBy]
+Person verification details::
+The `person[?].verifiedBy` property MUST be an object and MUST be present. The `person[?].verifiedBy` property MUST contain details about the _<<_identity_verifier,identity verifier>>_ and the identity verification process. This specification mentions at least three properties that MAY be used to represent the person verification details: `id`, `name`, and `proof`.
+
+[#vc-credentialsubject-person-verifiedBy-id]
+Identity verifier ID:::
+The `person[?].verifiedBy.id` MUST be present and MUST be a valid URI that, when dereferenced, MUST result in a proof of authenticity of the _<<_identity_verifier,identity verifier>>_. This proof of authenticity of the _<<_identity_verifier,identity verifier>>_ must not be confused with the proof of verification of the _<<_named_actor,named actor>>_
+
+[#vc-credentialsubject-person-verifiedBy-name]
+Identity verifier name:::
+The `person[?].verifiedBy.name` MUST be present and MUST not be empty. The `person[?].verifiedBy.name` property is the name of the _<<_identity_verifier,identity verifier>>_.
+
+[#vc-credentialsubject-person-verifiedBy-proof]
+Identity verification proof:::
+The `person.verifiedBy[?].proof` is an optional field. If present, it MUST be a valid URI that, when dereferenced, MUST result in the proof of authenticity of the  _<<_named_actor,named actor>>_, as per the verification process of the _<<_identity_verifier,identity verifier>>_. The content as well as the process are outside the scope of this specification.
+
+[#vc-credentialsubject-identities]
+Identities::
+The `identities` property is an optional field. If present, it MUST be an array containing information about the accounts that the _<<_named_actor,named actor>>_ requested to be bound with the _<<C2PA asset>>_ being described.
++
+.Example:
+[source,json]
+----
+"credentialSubject": {
+  ...
+  "identities": [
+    {
+      "type": "socialProvider",
+      "username": "username",
+      "uri": "https://twitter.com/username",
```

Comment:
Change to X.com? (in general, change all references of Twitter to X.

Reply:
I changed `Twitter` with `Example Social Network`. I hope that's ok. I thought that having an existing social provider might sound biased.

Thank you for the suggestion.